resources/lang/en/events/misc/general.json

```
@@ -2255,14 +2256,16 @@
       "desert"
     ],
     "season": ["any"],
+    "tags": ["clan_wide"],
     "weight": 20,
-    "event_text": "Tracks cross the border and then hurriedly cross out again, indicating the stranger didn't intrude intentionally.",
+    "event_text": "c_n finds tracks crossing the border and then hurriedly crossing out again, indicating the owner didn't intrude intentionally.",
```

```suggestion
    "event_text": "c_n finds tracks crossing the border and then hurriedly backtracking, indicating the owner didn't intrude intentionally.",
```

Not sure I'm a fan of the word "owner" tbh... but idk what I'd use instead. "Their creator" maybe?

---

Ehhh, we disagree here, I think "their creator" sounds weirder lolol, might just be a personal preference thing. i feel like i've seen "owner" used in this context pretty frequently.